src/apiCalls.js

```diff
@@ -0,0 +1,19 @@
+export const fetchMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => {
+        if(!response.ok) {
+          throw Error('Error fetching movies')
+        }
+        return response.json()
+      })
+  }
+
+export const fetchMovieInfo = (id) => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+        .then(response => {
+            if(!response.ok) {
+                throw Error('Error fetching movies')
+            } 
+            return response.json()
+        })
```

Comment:
I'm noticing this looks pretty similar to your other fetch - maybe you could refactor to use the URL as an argument and dry these up into one?